pages/changelog/2025-08-26-enhanced-observation-types.mdx

```diff
@@ -0,0 +1,38 @@
+---
+date: 2025-08-26
+title: Additional Observation Types for More Meaningful Span Context
+description: New observation types including Agent, Tool, Chain, Retriever, Evaluator, Embedding, and Guardrail provide semantic meaning to your traces.
+author: Nimar
+ogImage: /images/changelog/2025-08-26-additional-observation-types.png
```

Comment:
this needs to be 16:9 to render nicely on changelog index page, when using cleanshot you can set the output aspect ratio, lmk if you need help here